_includes/portfolio.html

```diff
@@ -0,0 +1,79 @@
+{% if include.portfolio %}
+	{% assign portfolio = include.portfolio %}
+{% else %}
+	{% assign portfolio = page.portfolio.items %}
+{% endif %}
+{% if include.categories %}
+	{% assign categories = include.categories %}
+{% else %}
+	{% assign categories = page.portfolio.categories %}
+{% endif %}
+{% if page.portfolio.button %}
+	{% assign button = page.portfolio.button %}
+{% else %}
+	{% assign button = "Launch Site" %}
+{% endif %}
+{% if include.defaultThumbnail %}
+	{% assign defaultThumbnail = include.defaultThumbnail %}
+{% elsif page.portfolio.defaultThumbnail %}
+	{% assign defaultThumbnail = page.portfolio.defaultThumbnail %}
+{% else %}
+	{% assign defaultThumbnail = "/style/images/defaultThumbnail.png" %}
+{% endif %}
+
+<div class="portfolio-wrapper showcase">
+	<ul class="filter">
+		<li><a class="active" href="#" data-filter="*">All</a></li>
+		{% for category in categories %}
+			<li><a href="#" data-filter=".{{ category[0] }}">{{ category[1] }}</a></li>
+		{% endfor %}
+	</ul>
+	<ul class="items col4">
+		{% for item in portfolio %}
+		<li class="item {{ item.categories|join: ' ' }}{% unless item.thumbnail %} defaultThumbnail{% endunless %}{% unless item.images %} columns{% endunless %}" ata-callback="callPortfolioScripts();" data-detailcontent='
+			<div class="content">
+				{% assign num_images = item.images|size %}
+				<div {% if num_images > 1 %}id="myCarousel" class="carousel slide"{% else %}class="single"{% endif %}>
```

Comment:
I can change it if you want, but it seems unnecessary. :confused:

As I said above, this entire section is contained in an attribute of the `li` tag named `data-detailcontent`.  It's rendered later on (and at a different position on the page) by the Javascript.  Because this section is created and destroyed by the script, there's only ever one ID of "myCarousel" at any given moment.

But, if you'd like, I can still change it. :wink: